README.md

```diff
@@ -3,15 +3,17 @@
 LNDK is a standalone daemon that connects to [LND](https://github.com/lightningnetwork/lnd) (via its grpc API) that aims to implement [bolt 12](https://github.com/lightning/bolts/pull/798) functionality _externally_ to LND. LNDK leverages the [lightning development kit](https://github.com/lightningdevkit/rust-lightning) to provide functionality, acting as a thin "shim" between LND's APIs and LDK's lightning library.
 
 Project Milestones:
+
```

Comment:
These newlines after headings have no effect on the rendered markdown, but it does make the source more readable :+1: